azure-pipelines.yml

```
@@ -40,8 +40,7 @@ jobs:
   - script: |
       export PATH=$HOME/miniconda3/bin:$PATH
       sudo apt-get install -y libc6-dev-i386
-      ci/incremental/install_miniconda.sh
-      ci/incremental/setup_conda_environment.sh
+      ci/setup_env.sh
     displayName: 'Set up environment'
```

```suggestion
    displayName: 'Setup environment and build pandas'
```

For consistency with the other azure build